russian/wca-regulations.md

```diff
@@ -464,25 +467,14 @@
     - C1b) Во время сборки участник может дотрагиваться до куба только одной рукой. Штраф: дисквалификация попытки (DNF).
         - C1b2) Если происходит повреждение головоломки и участник решает починить её, он должен чинить её только одной рукой (но обязательно той, которой он собирал). Штраф: дисквалификация попытки (DNF).
         - C1b3) Если происходит повреждение головоломки и ее элементы кратковременно задевают участника помимо его воли, то по решению судьи это за касание не считается.
+        - C1b4) Во время сборки участник может держать головоломку на поверхности стола для удобства (см. [Положение 7f1d](regulations:regulation:7f1d)).
     - C1c) Во время сборки, как только участник коснулся головоломки одной рукой, он не может касаться её другой рукой. Штраф: дисквалификация попытки (DNF).
 
 
-## <article-D><feet><solvingwithfeet> Статья D: Сборка ногами
-
-- D1) Процедура такая же, как и в [статье A](regulations:article:A) (скоростная сборка). Ниже описаны дополнительные положения, которые замещают соответствующие положения в [статье A](regulations:article:A).
-    - D1a) Во время попытки участник должен сидеть на стуле, на полу или стоять.
-    - D1b) Во время попытки участник должен использовать только ступни и поверхность пола для воздействия на головоломку. Штраф: дисквалификация попытки (DNF).
-- D3) Начало сборки:
-    - D3a) Участник ставит ступни на сенсоры таймера.
-    - D3b) Участник убирает ступни с таймера, чтобы начать сборку.
-- D4) Окончание сборки:
-    - D4a) Участник ставит ступни на сенсоры таймера, останавливая таймер.
-
-
 ## <article-E><fewest-moves><fewestmovessolving> Статья E: Сборка на количество ходов
 
 - E2) Процедура сборки на количество ходов:
-    - E2a) Судья раздаёт скрамблы и бумагу всем участникам, затем запускает секундомер и объявляет начало попытки.
+    - E2a) Судья раздаёт скрамблы и бумагу всем участникам, затем запускает секундомер и объявляет начало попытки. Судья запускает секундомер и говорит "Начали".
```

Comment:
You are repeating yourself here, I think it should go like this:
```suggestion
    - E2a) Судья раздаёт скрамблы и бумагу всем участникам. Судья затем запускает секундомер и говорит "Начали".
```

Reply:
Ok. Thanks!